charts/hivemq-platform/tests/services/hivemq_rest_api_services_test.yaml

```diff
@@ -147,3 +147,26 @@ tests:
       - notFailedTemplate: {}
       - hasDocuments:
           count: 1
+
+  - it: with a REST API service enabled and HiveMQ listener name enabled, validation fails
+    set:
+      services:
+        - type: rest-api
+          exposed: true
+          containerPort: 8888
+          hivemqListenerName: foobar
+    asserts:
+      - failedTemplate:
+          errorPattern: Service type `rest-api` with container port `8888` is using `hivemqListenerName` value. HiveMQ listener names are only supported by MQTT and WebSocket services
```

Comment:
Container port?

Reply:
Same comment is applicable here: https://github.com/hivemq/helm-charts/pull/403#discussion_r1866225099